ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -1439,7 +1439,8 @@ private letOrUse{ROLE, Getter}:
   LET<@ROLE, @Getter> |
   USE<@ROLE, @Getter> |
   LET_BANG<@ROLE, @Getter> |
-  USE_BANG<@ROLE, @Getter>;
+  USE_BANG<@ROLE, @Getter> |
+  AND_BANG<@ROLE, @Getter>;
```

Comment:
What if there is an `and` keyword? We should support getting it from bindings too.

Reply:
Thanks, I kinda forgot about using "and" with recursive functions. Will add and test.

Reply:
Perhaps there's already one, but please check it indeed!